Formula/android-ndk.rb

```diff
@@ -46,4 +45,10 @@ def caveats; <<-EOS.undent
       #{prefix}/docs
     EOS
   end
+
+  test do
+    (testpath/"test.c").write("int main() { return 0; }")
+    cc =  `/usr/local/opt/android-ndk/ndk-which gcc`.strip
```

Comment:
Thanks for the feed back and the pointer. New commit incorporates this suggestion.